plutus-ledger/plutus-ledger.cabal

```diff
@@ -1,4 +1,4 @@
-cabal-version: 2.2
+cabal-version: 3.0
```

Comment:
Needed in order to include generators from `cardano-api` in cabal with `cardano-api:gen -any`.